contracts/coordinator/src/client/index.ts

```diff
@@ -0,0 +1,892 @@
+     * @return  Transaction hash.
+     */
+    @decorators.asyncZeroExErrorHandler
+    public async marketSellOrdersFillOrKillAsync(
+        orders: Order[],
+        takerAssetFillAmount: BigNumber,
+        signatures: string[],
+        txData: TxData,
+        sendTxOpts: SendTransactionOpts = { shouldValidate: true },
+    ): Promise<string> {
+        return this._marketBuySellOrdersAsync(
+            ExchangeFunctionName.MarketSellOrdersFillOrKill,
+            orders,
+            takerAssetFillAmount,
+            signatures,
+            txData,
+            sendTxOpts,
+        );
+    }
+
+    /**
+     * No throw version of marketSellOrdersAsync
+     * @param   orders               An array of orders to fill.
+     * @param   takerAssetFillAmount Taker asset fill amount.
+     * @param   signatures           Signatures corresponding to the orders.
+     * @param   txData               Transaction data. The `from` field should be the user Ethereum address who would like
+     *                               to fill these orders. Must be available via the Provider supplied at instantiation.
+     * @param   sendTxOpts           Optional arguments for sending the transaction.
+     * @return  Transaction hash.
+     */
+    @decorators.asyncZeroExErrorHandler
+    public async marketSellOrdersNoThrowAsync(
+        orders: Order[],
+        takerAssetFillAmount: BigNumber,
+        signatures: string[],
+        txData: TxData,
+        sendTxOpts: SendTransactionOpts = { shouldValidate: true },
+    ): Promise<string> {
+        return this._marketBuySellOrdersAsync(
+            ExchangeFunctionName.MarketSellOrdersNoThrow,
+            orders,
+            takerAssetFillAmount,
+            signatures,
+            txData,
+            sendTxOpts,
+        );
+    }
+
+    /**
+     * Match two complementary orders that have a profitable spread.
+     * Each order is filled at their respective price point. However, the calculations are
+     * carried out as though the orders are both being filled at the right order's price point.
+     * The profit made by the left order goes to the taker (who matched the two orders).
+     * @param leftOrder              First order to match.
+     * @param rightOrder             Second order to match.
+     * @param leftSignature          Proof that order was created by the left maker.
+     * @param rightSignature         Proof that order was created by the right maker.
+     * @param   txData               Transaction data. The `from` field should be the user Ethereum address who would like
+     *                               to fill these orders. Must be available via the Provider supplied at instantiation.
+     * @param   sendTxOpts           Optional arguments for sending the transaction.
+     * @return  Transaction hash.
+     */
+    @decorators.asyncZeroExErrorHandler
+    public async matchOrdersAsync(
```

Comment:
Order matching is not supported by the coordinator server.

Reply:
chatted on Slack, `matchOrders` and `matchOrdersWithMaximalFill` are supported, just not the batch versions. https://github.com/0xProject/0x-monorepo/blob/development/contracts/coordinator/contracts/src/MixinCoordinatorApprovalVerifier.sol#L113-L114